src/cephadm/cephadm

```diff
@@ -565,6 +565,10 @@ def make_log_dir(fsid, uid=None, gid=None):
     makedirs(log_dir, uid, gid, LOG_DIR_MODE)
     return log_dir
 
+def make_var_run(fsid, uid, gid):
+    call_throws(['install', '-d', '-m0770', '-o', str(uid), '-g', str(gid),
+                 '/var/run/ceph/%s' % fsid])
```

Comment:
Path is similar to `LOCK_DIR` ... 

https://github.com/ceph/ceph/blob/e4c1c0e610209fe6423a063bed73a9e48fcb266b/src/cephadm/cephadm#L6

Should the `cephadm` lock files move to this location too?  
```
/var/run/ceph/<fsid>/cephadm.lock
```

Reply:
i put the cephadm ones separate since (1) you can't delete them when you're done without introducing races and we want to be able to clean up /var/run/ceph/$fsid probably, and (2) it seems possible that we'd want to have cross-cluster locks (maybe c-v? not sure).  Also just to have teh cephadm ones separate from the ceph admin socket files that normally live in /var/run/ceph